### ISSUE TITLE
[CI/CD]: Review coverage of accessibility checks in Education apps' end-to-end tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7348]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Applications **must** have thorough end-to-end tests that run in our continuous integration/continuous deployment (CI/CD) pipeline. These tests should include thorough axe checks. While auditing the `/education` apps folder, I noticed we could be opening one or two accordions on the review form page before running the axe check. I'd like the front-end engineering team to review the applications, and update the tests to review this hidden content. Definition of done in acceptance criteria below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Environment
* `vets-website/src/applications/edu-benefits/0993/tests`
* `vets-website/src/applications/edu-benefits/0994/tests`
* `vets-website/src/applications/edu-benefits/1990/tests`
* `vets-website/src/applications/edu-benefits/1990e/tests`
* `vets-website/src/applications/edu-benefits/1990n/tests`
* `vets-website/src/applications/edu-benefits/1995/tests`
* `vets-website/src/applications/edu-benefits/5495/tests`
* `$ yarn test:e2e src/applications/edu-benefits/<APP_NAME>/tests`

## Acceptance Criteria

**Definition of done:**
- [ ] Front-end team member(s) have reviewed end-to-end tests for axe checks
- [ ] Open 1 or 2 accordions on the review page (last page before submitting) and run the axe check
- [ ] FE team has consulted with accessibility specialist in cases where there are high numbers of modals, accordions, other hidden content that could slow down e2e test runs.
- [ ] No axe `violations` appear in the plugin console. These will break the CI/CD build.

## WCAG or Vendor Guidance (optional)

* [Custom axeCheck helper method](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js)
* [VSP guidance on writing end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing)

**Comment by 1Copenut** [comment]
@mattpointzxer0 If the team has moved to use the Cypress forms testing, would you mind commenting here and at include me? The Cypress forms tester has axe checks built in and would be a good way to close this issue without additional work.

**Comment by 1Copenut** [comment]
@mattpointzxer0 I'd like to revisit this issue when things are settled a bit with COVID related work.
